src/main/java/io/aiven/kafka/connect/opensearch/BulkProcessor.java

```diff
@@ -363,11 +391,19 @@ public BulkResponse call() throws Exception {
             }
         }
 
+        private void reportToDlq(final String errorMessage, final SinkRecord batchRecord) {
```

Comment:
We really can't assume that this will end up in a DLQ topic :)
```suggestion
        private void sendToReporter(final String errorMessage, final SinkRecord batchRecord) {
```
(feel free to change the name--but nothing with DLQ in it!)